### ISSUE TITLE
New color picker improvements, first round

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2679]
I'd like to propose a first round of improvements to the new color picker recently refactored in #2406. Most notably:
- don't use aria-current to indicate the currently selected button; instead, use aria-pressed as already done for the block alignment buttons. These controls are a set of buttons where only one button can be activated at a time, so the best option is to communicate they're "toggle buttons" and that's what aria-pressed does. In some screen readers, activating/deactivating a button that uses aria-pressed makes the screen reader also announce "select color..." and "deselect color..."
- fix invalid code: divs inside buttons are invalid
- add missing aria-label on the remove color button
- make the Custom color picker toggle a real toggle that opens/closes and uses aria-expanded
- fix the focus style on the Custom color picker

**Comment by afercia** [comment]
The color picker has room for further usability and a11y improvements, some of them were quickly discussed with @mtias and will open a new issue to discuss them.

**Comment by 00travelgirl00** [comment]
@afercia I could also not use the open custom color picker with the keyboard. So I think the whole picker is not a11y, or?
